README.md

```diff
@@ -2,42 +2,41 @@
+Like we stated before, **we don't think that frameworks are evil**. They are the most powerful tools to learn how to code without them. This is because in order to know how and when you should use one particular framework you have to understand how frameworks work. Having clear opportunities and constraints of each framework it will be easier to make the right choice, also it will be clear when you don't need one. Furthermore, understanding how frameworks work behind the scenes will deepen your awareness about vanilla javascript. Every framework is written in vanilla javascript and it is very generic. Knowing the frameworks rationale you will able to write better specific code. For example, a rendering algorithm ad hoc for your application without depending on a framework.
 
 Nevertheless, we think that frameworks should be used responsibly. Remembering that every tool has some kind of tradeoff.
 
 ## What Frameworkless Movement is not
 
-Frameworkless **does not mean that we have to (or we want) build our own framework**. But it means taking a conscious decision about if we want to get help to do our own job from tools already exist and in what way.
+Frameworkless **does not mean that we have to (or we want to) build our own framework**. It means taking a conscious decision about if we want to get help to do our own job from tools already in existence and in what way.
 
-Frameworkless **does not mean "reinvent the wheel"**. There are a lot of "tools" (frameworks, libraries but also StackOverflow) that solve known and common problems in a great way. Frameworkless means, however, don't assume that in the software I'm writing I can afford to delegate this responsibility and depends on other (library for example) to solve a specific problem. Or worse: don't reinvent the wheel and bind hand and foot to a whole framework even if I don't really need.
+Frameworkless **does not mean "reinvent the wheel"**. There are a lot of "tools" (frameworks, libraries but also StackOverflow) that solve known and common problems in a great way. Frameworkless means, however, not to assume that in the software I'm writing I can afford to delegate this responsibility and depend on other software (a library for example) to solve a specific problem. Or worse: don't reinvent the wheel and bind hand and foot to a whole framework even if I don't really need it.
 
-Frameworkless **does not mean that you must develop all software by yourself without a framework** in every context and in every situation, thinking that this is more secure and more performant ([Not Invented Here syndrome](https://en.wikipedia.org/wiki/Not_invented_here#In_computing)), far from it.
+Frameworkless **does not mean that you must develop all software by yourself without a framework** in every context and in every situation, thinking that this is more secure and more high performance ([Not Invented Here syndrome](https://en.wikipedia.org/wiki/Not_invented_here#In_computing)), far from it.
```

Comment:
"this is more secure and more high performance" Could be changed in "more secure and with higher perfomances"?

Reply:
high perfomance is an adjective... I am not 100% how you write it, maybe with a - between the two words.

But if we go with your advice it should be in singular: performance i think

Reply:
It seems that you need a dash here. Can you update it? https://www.macmillandictionary.com/dictionary/british/high-performance

Reply:
@francesco-strazzullo Done!